packages/platform-sdk-xlm/src/services/message.ts

```diff
@@ -14,7 +14,7 @@ export class MessageService implements Contracts.MessageService {
 
 	public async sign(input: Contracts.MessageInput): Promise<Contracts.SignedMessage> {
 		try {
-			const privateKey: string = StellarHDWallet.fromMnemonic(BIP39.normalize(input.mnemonic)).getSecret(0);
+			const privateKey: string = StellarHDWallet.fromMnemonic(BIP39.normalize(input.mnemonic!)).getSecret(0);
```

Comment:
This needs a proper check that the mnemonic is set now that it isn't enforced by TypeScript.